projects/addon-charts/components/arc-chart/arc-chart.template.html

```diff
@@ -14,9 +14,11 @@
         d="M -70 70 A 100 100 0 1 1 70 70"
     />
     <path
+        #arc
```

Comment:
nit: don't you want to move it on svg to apply hover effect on whole arc ?

Reply:
I tried both and it felt better to highlight only when you hover over the filled part.